apps/prairielearn/src/sync/fromDisk/courseInstances.ts

```diff
@@ -64,7 +64,8 @@ function getParamsForCourseInstance(courseInstance: CourseInstanceJson | null |
     self_enrollment_enabled: courseInstance.selfEnrollment.enabled,
     self_enrollment_enabled_before_date: courseInstance.selfEnrollment.beforeDate,
     self_enrollment_use_enrollment_code: courseInstance.selfEnrollment.useEnrollmentCode,
-    self_enrollment_restrict_to_institution: courseInstance.selfEnrollment.restrictToInstitution,
+    publishing_start_date: courseInstance.publishing?.startDate ?? null,
+    publishing_end_date: courseInstance.publishing?.endDate ?? null,
```

Comment:
IMO we should start syncing `modern_publishing` as well in this PR, even if it's a sync error to enable it.